accio-main/src/main/java/io/accio/main/connector/duckdb/DuckDBMetadata.java

```diff
@@ -155,41 +178,140 @@ public String getPgCatalogName()
     }
 
     @Override
-    public String handlePgType(String type)
+    public void reload()
     {
-        if (type.startsWith("_")) {
-            return format("%s[]", handlePgType(type.substring(1)));
-        }
-        else if (!DuckdbTypes.getDuckDBTypeNames().contains(type)) {
-            return "VARCHAR";
-        }
-        return type;
+        // Create client success first to ensure the sql is legal, then close the old one
+        DuckdbClient duckdbClient = buildDuckDBClient();
+        close();
+        this.duckdbClient = duckdbClient;
```

Comment:
It's acceptable for now. However, there're a potential issue about if the init SQL load large data to DuckDB, it could cost 200% memory system needed when user try to reload client.

Reply:
You are right, but there is no other way to make sure that the SQL is running correctly.

Reply:
I prefer to close the original one firstly. The reload only is triggered when `put` initSql, `put` session sql and some reload-required configs been changed. It means user want to discard the original one or expects the DB will be reload.  I don't think we need to keep the old one to be the backup.